### PR TITLE
14 correct dropdown

### PR DESCRIPTION
**Pull request by ECCresym**
Drop down menu has been corrected. 
2 possible improvements: 
-Add one page for each model category 
-Fully automatise the drop down menu

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -9,3 +9,5 @@ _site
 
 # Bundle
 vendor
+Gemfile.lock
```
Comment (matoubongrain) [inline .gitignore:12]: could you remove the gemfile and notes.md from thhis file?

File: Gemfile.lock
Comment (matoubongrain) [inline Gemfile.lock:1]: i don't think this file should be part of the code, as it is automatically generated on your computer.

Comment (ECCresym) [reply Gemfile.lock:1]: It has been removed.

File: .gitignore (hunk 1 above)
Comment (ECCresym) [reply .gitignore]: Done.

**Review by matoubongrain — APPROVED** [review]

**Pull request merged**